src/server.ts

```diff
@@ -1,8 +1,19 @@
 import * as express from 'express';
+import * as bodyParser from 'body-parser';
+import { emailRouter } from './routes/email';
+import { authRouter } from './routes/auth';
+import { ensureAuthenticated } from './utils/auth/auth-utils';
 
 const PORT = process.env.PORT || 4000;
 const server = express();
 
+// parse application/json
+server.use(bodyParser.json());
+server.use(bodyParser.urlencoded({ extended: true }));
+
+server.use('/', authRouter);
```

Comment:
I would say `server.use('/auth', authRouter);` here rather than in the actual auth router, its a little misleading